tests/arp/test_arpall.py

```diff
@@ -0,0 +1,175 @@
+    int_facts = duthost.interface_facts()['ansible_facts']
+
+    ports = list(sorted(mg_facts['minigraph_ports'].keys(), key=lambda item: int(item.replace('Ethernet', ''))))
+
+    # Select port index 0 & 1 two interfaces for testing
+    intf1 = ports[0]
+    intf2 = ports[1]
+    logging.info("Selected ints are {0} and {1}".format(intf1, intf2))
+
+    intf1_indice = mg_facts['minigraph_port_indices'][intf1]
+    intf2_indice = mg_facts['minigraph_port_indices'][intf2]
+
+    po1 = get_po(mg_facts, intf1)
+    po2 = get_po(mg_facts, intf2)
+    
+    try:
+        # Make sure selected interfaces are not in portchannel
+        if po1 is not None:
+            duthost.shell('config portchannel member del {0} {1}'.format(po1, intf1))
+            duthost.shell('config interface startup {0}'.format(intf1))
+        
+        if po2 is not None:
+            duthost.shell('config portchannel member del {0} {1}'.format(po2, intf2))
+            duthost.shell('config interface startup {0}'.format(intf2))
+        
+        # Change SONiC DUT interface IP to test IP address
+        duthost.shell('config interface ip add {0} 10.10.1.2/28'.format(intf1))
+        duthost.shell('config interface ip add {0} 10.10.1.20/28'.format(intf2))
+
+        # TODO: Use wait_until to replace this
+        if (po1 is not None) or (po2 is not None):
+            time.sleep(40)
+
+        # Copy test files
+        ptfhost.copy(src="ptftests", dest="/root")
+
+        yield duthost, ptfhost, int_facts, intf1, intf2, intf1_indice, intf2_indice
+    finally:
+        # Recover DUT interface IP address
+        # TODO: Convert reload_config.yml, use below command to do the recover for now
+
+        if po1 is not None:
+            duthost.shell('config portchannel member add {0} {1}'.format(po1, intf1))
+            duthost.shell('config interface startup {0}'.format(intf1))
+        
+        if po2 is not None:
+            duthost.shell('config portchannel member add {0} {1}'.format(po2, intf2))
+            duthost.shell('config interface startup {0}'.format(intf2))
+
+        # TODO: Use wait_until to replace this
+        if (po1 is not None) or (po2 is not None):
+            time.sleep(40)
+
+        duthost.shell('config interface ip add {0} 10.10.1.2/28'.format(intf1))
+        duthost.shell('config interface ip add {0} 10.10.1.20/28'.format(intf2))
+
+def test_arp_unicast_reply(common_setup_teardown):
+    duthost, ptfhost, int_facts, intf1, intf2, intf1_indice, intf2_indice = common_setup_teardown
+
+    # Start PTF runner and send correct unicast arp packets
+    clear_dut_arp_cache(duthost)
+    params = {
+        'acs_mac': int_facts['ansible_interface_facts'][intf1]['macaddress'],
+        'port': intf1_indice
+    }
+    log_file = "/tmp/arptest.VerifyUnicastARPReply.{0}.log".format(datetime.now().strftime("%Y-%m-%d-%H:%M:%S"))
+    ptf_runner(ptfhost, '/tmp', "arptest.VerifyUnicastARPReply", '/root/ptftests', params=params, log_file=log_file)
```

Comment:
the second parameters should be "ptftests" instead of '/tmp'

Reply:
Yes! You are right!